.github/workflows/R-CMD-check.yaml

```diff
@@ -43,6 +45,7 @@ jobs:
         run: Rscript -e "install.packages('INLA',repos=c(getOption('repos'),INLA='https://inla.r-inla-download.org/R/stable'), dep=TRUE)"
 
       - uses: r-lib/actions/setup-r-dependencies@v1
+        env:
```

Comment:
@jimhester is this where the CRAN variable should go?

Reply:
Thanks for catching this, it needs to go in `setup-r` as, that is where the user level `~/.Rprofile` is setup, which is when the CRAN envvar is checked.

I put this in the wrong place initially, but forgot to remove the stray `env:`